src/core/common/random_manager.cpp

```diff
@@ -190,8 +191,19 @@ int RandomManager::Entropy::HandleMbedtlsEntropyPoll(void *         aData,
 
 void RandomManager::CryptoCtrDrbg::Init(void)
 {
+    int rval;
+
     mbedtls_ctr_drbg_init(&mCtrDrbg);
-    mbedtls_ctr_drbg_seed(&mCtrDrbg, mbedtls_entropy_func, RandomManager::GetMbedTlsEntropyContext(), nullptr, 0);
+
+    rval =
+        mbedtls_ctr_drbg_seed(&mCtrDrbg, mbedtls_entropy_func, RandomManager::GetMbedTlsEntropyContext(), nullptr, 0);
+
+    if (rval != 0)
+    {
+        otLogCritMbedTls("Can't seed the CTR DRBG");
```

Comment:
nit:
```suggestion
        otLogCritMbedTls("Failed to seed the CTR DRBG");
```
I think "fail" is a more common search word when analyzing logs.

Reply:
I agree.